_gsocproposals/2019/proposal_AstroLabSparkfits.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Apache Spark: migrating FITS Data Source API to V2
+layout: gsoc_proposal
+project: AstroLab
+year: 2019
+organization: LAL
+---
+
+## Description
+
+High Energy Physics & Astrophysics experiments typically describe their data with complex data structures that are stored into structured file formats such as for example Hierarchical Data Format 5 (HDF5), ROOT files, or Flexible Image Transport System (FITS). For example, the FITS file format has been introduced more than thirty years ago with the requirement that developments to the format must be backward compatible. This data format has been used successfully to store and manipulate the data of a wide range of astrophysical experiments over the years, hence it is considered as one of the possible data format for future surveys like the Large Synoptic Survey Telescope (LSST) which will produce PB of data over the next years.
```

Comment:
@JulienPeloton Do you think it makes sense to add a link to FITS documentation since it is the most relevant data format for this project?